### DISCUSSION TITLE
Add another user to CommandHandler.filters in runtime

### DISCUSSION THREAD

**Discussion by ekiauhce** [python-telegram-bot/python-telegram-bot#2326] [Q&A] [answered]
I want to achieve behavior like this:
1. Call `/add_admin 123456` command where 123456 is user_id
2. In `add_admin` function somehow update `foo_handler.filters` (foo_handler is CommandHandler object, wich handle `/foo` command)
3. And now user with user_id = 123456 should have access to `/foo` command

**Comment by Bibo-Joshi** [comment, marked as answer]
Hey `Filters.user` has `add/remove_user` methods which can be used to (dis)allow users at runtime. You may also be interested in the `Roles` setup over at ptbcontrib, which gives a full blown role based access management.

**Reply by ekiauhce** [reply]
Thank you for reply! I was looking for add/remove methods. Also i found out, that i should create particular instance of `Filter` class. Because `CommandHandler.filters` has type `MergedFilter` for me.
